core/main/src/firebolt/handlers/localization_rpc.rs

```diff
@@ -335,91 +312,6 @@ impl LocalizationServer for LocalizationImpl {
         )
         .await
     }
-
-    async fn timezone_set(
-        &self,
-        _ctx: CallContext,
-        set_request: TimezoneProperty,
-    ) -> RpcResult<()> {
-        let resp = match self
-            .platform_state
-            .get_client()
-            .send_extn_request(DeviceInfoRequest::GetAvailableTimezones)
```

Comment:
Are we now also able to clean up `DeviceInfoRequest` to get rid of `SetTimeZone`, `GetTimezone`, `GetAvailableTimezones`, `GetTimezoneWithOffset`, etc or are they still in use? Same question for `ExtnResponse::AvailableTimezones`, `ExtnResponse::TimezoneWithOffset`, etc.

Reply:
@pahearn73 : What about the `DeviceInfoRequest` timezone usage here 
https://github.com/rdkcentral/Ripple/blob/main/core/main/src/service/context_manager.rs#L109